spikewidgets/widgets/multicompgraphwidget/multicompgraphwidget.py

```diff
@@ -86,7 +86,7 @@ def plot_multicomp_agreement(multi_sorting_comparison, plot_type='pie',
 
 
 def plot_multicomp_agreement_by_sorter(multi_sorting_comparison, plot_type='pie',
-                                       cmap='YlOrRd', figure=None, ax=None):
+                                       cmap='YlOrRd', figure=None, axes=None, show_legend=True):
```

Comment:
the nice thing about the MultiWidget is that it can create gridspec subplots from an existing single axis. I think it is better to keep it that way, or at least keep both args.

Reply:
Yes, but I needed it here to fit these into a publishable figure. My solution is not the most elegant I admit, but it does the job. Maybe allow both?

Reply:
Ok then when you're done with the changes I'll add it to the BaseMultiWidget

Reply:
Not 100% happy with current solution, but ok for now?